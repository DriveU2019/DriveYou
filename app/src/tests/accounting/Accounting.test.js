/*
 * Copyright 2019 [dage] @DriveYou
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/*
 * --------------------------------------------------------------------------------
 * Description: Test If Component Render Correct!
 *        TODO:
 * --------------------------------------------------------------------------------
 */

import React from 'react';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Accounting from '../../components/accounting/Accounting';

configure({ adapter: new Adapter() });

describe('Accounting main Component', () => {
  it('should render correctly', () => {
    const component = shallow(<Accounting />);

    expect(component).toMatchSnapshot();
  });
});
