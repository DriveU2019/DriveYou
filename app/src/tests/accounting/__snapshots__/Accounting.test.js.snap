// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Accounting main Component should render correctly 1`] = `
<div>
  ACCOUNTING
</div>
`;
